auth/jwt.go

```diff
@@ -25,33 +26,33 @@ var (
 )
 
 // GetJWTField gets the JWT from a context and returns the specified field
-func GetJWTField(ctx context.Context, field string, keyfunc jwt.Keyfunc) (string, error) {
-	token, err := getToken(ctx, keyfunc)
+func GetJWTField(ctx context.Context, tokenField, key string, keyfunc jwt.Keyfunc) (string, error) {
```

Comment:
how would you feel about if, instead of breaking the contract, adding an overload (`GetJWTFieldWithTokenName` or something) that adds the functionality?